### PR TITLE
Allow specifying Ingress rules with http.paths with no path.

### PR DESCRIPTION
**Pull request by xa4a**
These can be rules that map different hostnames to backends, where each
backend handles all paths, so has no `path` field. For example:

```
spec:
  rules:
  - host: a.example.com
    http:
      paths:
      - backend:
          serviceName: service-a
          servicePort: web
  - host: b.example.com
    http:
      paths:
      - backend:
          serviceName: service-b
          servicePort: web
```

Currently this fails with:

```
RUNTIME ERROR: Field does not exist: path
	kube-libsonnet/kube.libsonnet:588:26-32	thunk from <object <anonymous>>
```

### CODE REVIEW

**Comment by jjo** [comment]
@xa4a : Thanks for fixing, looks great -- mind adding some tests for this case? I'd be matter of just adding a new field for it to tests/test-simple-validate.pass.jsonnet

**Review by jbianquetti-nami — APPROVED** [review]
SGTM, thanks!

**Comment by xa4a** [comment]
Is there anything blocking this PR from being merged?

**Comment by xa4a** [comment]
Looks like I still need an approval from @dbarranco

**Review by dbarranco — APPROVED** [review]
Sorry for the late approval, thanks a lot!

**Comment by jjo** [comment]
wohhOO thanks @dbarranco :hugs:

**Pull request merged**
